sw/supervision/python/app_settings.py

```diff
@@ -0,0 +1,26 @@
+from PyQt5.QtWidgets import *
```

Comment:
add license headers to all python files ?

Reply:
I added a short 2 line notice. Not a big fan of huge license headers.